easybuild/easyconfigs/0/3d-dna/3d-dna-180922-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'Tarball'
+
+name = '3d-dna'
+version = '180922'
+versionsuffix = '-Python-%(pyver)s'
+local_githash = '529ccf46599825b3047e58a69091d599e9858a19'
+
+homepage = 'https://github.com/theaidenlab/3d-dna'
+description = """3D de novo assembly (3D DNA) pipeline"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
Do you really need to use `foss` here, none of the dependencies require it?

Reply:
Should this then be the SYSTEM toolchain or GCC or GCCCore ?

Reply:
```suggestion
toolchain = {'name': 'GCC', 'version': '8.2.0'}